core/src/main/scala/models.scala

```diff
@@ -3,12 +3,7 @@ package wiro
 import io.circe._
 
 package object models {
-  case class ClientConfig(
-    host: String,
-    port: Int
-  )
-
-  case class ServerConfig(
+  case class Config(
     host: String,
     port: Int
```

Comment:
I'd add `protocol` or at least the possibility to choose between `http` and `https`.